### DISCUSSION TITLE
allthemods10+9

### DISCUSSION THREAD

**Discussion by Leach2257** [AdvancedXRay/XRay-Mod#287] [Q&A] [answered]
will the ability to xray the allthemods specific ores like allthemodium vibranium and unobtainium be added?

**Comment by MichaelHillcox** [comment, marked as answer]
You can xray for any block in a pack

**Reply by Leach2257** [reply]
none of the above mentioned ore show up in the selection

**Reply by MichaelHillcox** [reply]
Then add it using the buttons on the right of the gui
